lua/languages/global/languages/css.lua

```diff
@@ -3,24 +3,15 @@
 
 local global = require("core.global")
 local languages_setup = require("languages.global.utils")
-local nvim_lsp = require("lspconfig")
 local nvim_lsp_util = require("lspconfig/util")
 local lsp_signature = require("lsp_signature")
 local default_debouce_time = 150
-local lsp_installer = require("nvim-lsp-installer")
-local lsp_installer_servers = require("nvim-lsp-installer.servers")
 
 local language_configs = {}
 
 language_configs["lsp"] = function()
-    local function start_cssls()
-        nvim_lsp.cssls.setup {
-            cmd = {
-                "node",
-                global.lsp_path ..
-                    "lsp_servers/vscode-langservers-extracted/node_modules/.bin/vscode-css-language-server",
-                "--stdio"
-            },
+    local function start_cssls(server)
+        server:setup {
```

Comment:
The `server` argument here is an instance of `:h nvim-lsp-installer.Server`. It's recommended to set up servers through that function instead, as it will fill in the necessary lspconfig options for you (`cmd` in particular, but in some cases there are other necessary settings like `cmd_env` for the Haskell LSP).